.travis.yml

```diff
@@ -125,10 +136,9 @@ install:
         sqlalchemy
         mock
         nose
-        gdal
```

Comment:
I opened a separate issue to re-enable it (https://github.com/obspy/obspy/issues/1749) as I'd really like to get this PR merged.